core/src/test/java/org/opensearch/sql/expression/text/TextFunctionTest.java

```diff
@@ -377,6 +377,20 @@ void locate() {
         DSL.locate(missingRef, DSL.literal("hello"), DSL.literal(1))));
   }
 
+  @Test
+  void position() {
+    FunctionExpression expression = DSL.position(
+        DSL.literal("world"),
+        DSL.literal("helloworld"));
+    assertEquals(INTEGER, expression.type());
+    assertEquals(6, eval(expression).integerValue());
```

Comment:
you need a test that returns `0` (missing substring)

Reply:
Addressed in 9a6b7b5d8962cff8af410638da5151e9ee6a36b9